src/components/CourseForm/index.test.tsx

```diff
@@ -183,17 +459,40 @@ describe('Course Form Course Edit Tests', () => {
   const oneDayBeforeStart = new Date();
 
   const template = {
-    id: '',
-    name: '',
-    description: '',
-    summary: '',
-    tags: [],
-    maxStudents: 0,
-    createdById: '',
-    image: '',
+    id: '1234',
+    name: 'New course',
+    description: 'A test course',
+    summary: 'All you ever wanted to know about testing!',
+    tags: [{ id: '1', name: 'Testing' }],
+    maxStudents: 55,
+    createdById: '30',
+    image: 'http://test-image.com',
   };
 
```

Comment:
I suggest putting one 'const course = { id: '1234', createdById: '30', ... }' here because now it is copied to many different parts beneath this 'describe('Course Form Course Edit Tests',..)' test section.